routes/v1.js

```diff
@@ -313,7 +313,7 @@ function addRoutes(app, peliasConfig) {
       middleware.sendJSON
     ]),
     place: createRouter([
-      sanitizers.place.middleware,
+      sanitizers.place.middleware(peliasConfig.api),
```

Comment:
@missinglink, what do you think is our current thinking here? We can either add a parameter to all the middleware that expects an instance of the Pelias configuration, or potentially we could add a `const config = require('pelias-confg').generate().api` line to the middleware, right?

The first is technically more efficient from a code perspective (only a single instance of the config object is generated), but the second probably lowers the conceptual overhead (there's nothing to my knowledge that would prevent the middleware from being able to gather the config on its own) and allows for a cleaner interface with no parameters needed.

@blackmad since you were just in this code, can you confirm that we could, if we wanted, put a new `require` line in the middleware logic instead of passing it around a config object everywhere?

Reply:
autocomplete.js and search.js already passed this in, so this was bringing in the other two big sanitizers into same pattern